packages/cli/package.json

```diff
@@ -11,30 +11,23 @@
     "url": "git+https://github.com/ampproject/amp-toolbox.git",
     "directory": "packages/cli"
   },
-  "keywords": [
-    "amp-toolbox",
-    "cli"
-  ],
-  "files": [
-    "index.js",
-    "lib",
-    "bin"
-  ],
+  "keywords": ["amp-toolbox", "cli"],
+  "files": ["index.js", "lib", "bin"],
   "author": "AMPHTML Team",
   "license": "Apache-2.0",
   "bugs": {
     "url": "https://github.com/ampproject/amp-toolbox/issues"
   },
   "homepage": "https://github.com/ampproject/amp-toolbox/tree/main/packages/cli",
   "dependencies": {
-    "@ampproject/toolbox-cache-list": "2.10.1",
-    "@ampproject/toolbox-cache-url": "2.10.1",
-    "@ampproject/toolbox-core": "2.10.1",
-    "@ampproject/toolbox-linter": "2.10.1",
+    "@ampproject/toolbox-cache-list": "^2.11.0",
+    "@ampproject/toolbox-cache-url": "^2.11.0",
+    "@ampproject/toolbox-core": "^2.11.0",
```

Comment:
we'd prefer to use the hard pinned version `2.11.0`

Reply:
> we'd prefer to use the hard pinned version `2.11.0`

done